BootNodeDev/pathfinder
# Fix/deploy api as a server

## Description

This PR adds the following changes:

- Adds a package json and lock files to be able to install the needed dependencies of the server app version
  - Also, adds pm2 as a dependency to run the server with some commands
- Updates the default PORT used in the webservice

## How to test

- Build pathfinder (https://github.com/BootNodeDev/pathfinder#the-core-executable)
- Remember to install npm/node (>=v14)
- Install Dependencies
  - npm install
- Start Server
  - npm run prod
- Verify that app is running in port: 8000

## Example

![image](https://user-images.githubusercontent.com/13955827/180488883-d29149a1-83be-43e3-a5fb-52c7e5a005ef.png)
